docs/config/node.rst

```diff
@@ -205,6 +208,65 @@ Ports
   range is used. If this is set to an integer value it is considered as an
   explicit single port.
 
+Advanced TCP settings
+---------------------
+
+Any interface that uses TCP (PG/HTTP/transport) shares the following settings:
```

Comment:
relevant `TranportSettings` and `HttpTransportSettings` take their defaults from the `NetworkService` settings.